src/components/BurgerMenu/BurgerMenu.tsx

```diff
@@ -0,0 +1,108 @@
+import { Link, useLocation } from 'react-router-dom';
+import './BurgerMenu.scss';
+import { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import CrossImg from '../../images/icons/Cross.png';
+import FavoritesIcon from '../../images/icons/Favourites (Heart Like).svg';
+import CardIcon from '../../images/icons/Shopping bag (Cart).svg';
+
+export const BurgerMenu = () => {
+  const navLinks = [
+    { value: 'HOME', hreaf: '/home' },
+    { value: 'PHONES', hreaf: '/phones' },
+    { value: 'TABLETS', hreaf: '/tablets' },
+    { value: 'ACCESSORIES', hreaf: '/accessories' },
+  ];
+
+  const location = useLocation();
+  const cartLocation = location.pathname.includes('cart');
+  const favoritesLocation = location.pathname.includes('favorites');
+
+  const [isActive, setIsActive] = useState(false);
+
+  const toggleBurgerMenu = () => {
+    setIsActive(!isActive);
+  };
+
+  useEffect(() => {
+    if (isActive === true) {
```

Comment:
Note: work with dom is bad practice in react
```suggestion
    if (isActive) {
```